apps/builder/app/builder/features/style-panel/property-label.tsx

```diff
@@ -323,10 +323,12 @@ export const PropertyInlineLabel = ({
   description,
   properties,
   disabled,
+  syntax,
 }: {
   label: string;
   description: string;
-  properties: [StyleProperty, ...StyleProperty[]];
+  properties?: [StyleProperty, ...StyleProperty[]];
+  syntax?: string;
```

Comment:
maybe for this specific component we could relax properties type while keeping autocomplete.
<img width="423" alt="image" src="https://github.com/user-attachments/assets/4051378d-9bb9-451e-ac94-d06a7bf62c8d">

Reply:
Done, update the type and all the places where it is used 👍